libs/common/src/helpers/fee-record-corrections/map-correction-reasons-to-values.test.ts

```diff
@@ -0,0 +1,107 @@
+import { mapCorrectionReasonsToValues, correctionReasonsToValues } from './map-correction-reasons-to-values';
+import { FeeRecordEntityMockBuilder } from '../../test-helpers';
+import { RECORD_CORRECTION_REASON } from '../../constants';
+
+describe('map-correction-reasons-to-values', () => {
+  const feeRecord = new FeeRecordEntityMockBuilder().build();
+
+  describe('correctionReasonsToValues', () => {
+    describe(RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT, () => {
```

Comment:
For the test output to make more grammatical sense I think it would make sense for these describe blocks to me names like this:
```suggestion
    describe(`when the reason is ${RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT}`, () => {
```